app/migrations/versions/3b14e2aff4ea_.py

```diff
@@ -0,0 +1,32 @@
+"""Enforce non-nullable required fields for organisation
+
+Revision ID: 3b14e2aff4ea
+Revises: 7b3486815759
+Create Date: 2021-03-31 10:00:57.648366
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3b14e2aff4ea'
+down_revision = '7b3486815759'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('organisation', '_timezone',
```

Comment:
This migration fails on existing databases where there are existing `null` timezones   
![image](https://user-images.githubusercontent.com/2199643/113324027-009b1500-92ed-11eb-835d-db449c0f50e8.png)